CSSS/Kernel.cs

```diff
@@ -172,28 +167,34 @@ public bool PerformTasks()
             }
 
             // If we should shutdown
-            if (shouldShutdown)
+            if (config.CSSSProgramMode.HasFlag(Config.CSSSModes.Shutdown))
             {
                 // Run the shutdown command
                 var process = new System.Diagnostics.Process();
                 process.StartInfo.FileName = "shutdown";
                 switch (config.operatingSystemType)
                 {
                     case Config.OperatingSystemType.WinNT:
-                        // On Windows, we can specify the seconds
-                        // to delay the shutdown :D
-                        // So we shutdown in 15 seconds (should be suffficient time)
-                        process.StartInfo.Arguments = "-s -t 15";
+                        logger.Info("Operating System detected as WinNT - shutting down computer in 5 seconds");
+                        // On Windows, we can specify the seconds to delay the shutdown
+                        // So we shutdown in 15 seconds 
+                        // The logic behind this is that shutting down the program immediately
+                        // may cause issues if there is a longer lasting process and allows
+                        // time for the program to gracefully exit without being terminated
+                        // as the computer shutsdown
+                        process.StartInfo.Arguments = "-s -t 5";
                         break;
                     case Config.OperatingSystemType.Linux:
-                        // Linux doesn't let you specify seconds... >:(
+                        logger.Info("Operating System detected as WinNT - shutting down computer in 1 minute");
+                        // Linux doesn't let you specify seconds so it defaults to 1 minute
+                        // See above command for logic on shutdown
                         process.StartInfo.Arguments = "-h +1";

```

Comment:
I agree with your comments.

Let's set any shutdown command to have a 1 minute delay, that way there doesn't need to be different values for each OS.

If possible, could you also get the shutdown command to display a comment saying "Shutting down computer in 1 minute for image capture"

Additionally, could you move and combine the `logger.Info` statements to be before the `switch` statement, and to read "Attempting to shut down the computer in 1 minute" - that way it's in one location so you don't get any copy-paste issues(!)